html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <label for="email">Email</label>
+    <br />
+    <input type="text" id="email" name="name" required minlength="4" maxlength="18" size="10">
+    <br />
+    <label for="options">Why is coding cool?</label>
+    <br />
+    <select name="Select Options">
```

Comment:
make sure to add the id attribute, also you want to avoid having spaces in your attribute values like this. `select-options` would be better